lib/proc.c

```diff
@@ -722,6 +722,8 @@ proc_status(grn_ctx *ctx, int nargs, grn_obj **args, grn_user_data *user_data)
       GRN_OUTPUT_UINT64(0);
     }
   }
+  GRN_OUTPUT_CSTR("n_workers");
+  GRN_OUTPUT_INT32(grn_ctx_get_n_workers(ctx));
```

Comment:
We must increment `n_elements` too.

It's better that we append a new element to the last. Theoretically adding a new element into a map is backward compatible, but I'm not sure that all clients (for all output formats) are implemented as expected.

Reply:
I didn't notice `n_elements`. Thanks for the info.
I also moved `n_workers` to last.